packages/pluggableWidgets/accordion-native/src/Accordion.tsx

```diff
@@ -0,0 +1,141 @@
+import { createElement, ReactElement, ReactNode, useCallback, useEffect, useRef, useState } from "react";
+import { Animated, Easing, LayoutChangeEvent, Pressable, View, ViewStyle } from "react-native";
+import { Icon } from "mendix/components/native/Icon";
+import { exclude, flattenStyles } from "@mendix/piw-native-utils-internal";
+
+import { AccordionStyle, defaultAccordionStyle } from "./ui/Styles";
+import { AccordionProps } from "../typings/AccordionProps";
+
+declare interface GlyphIcon {
+    readonly type: "glyph";
+    readonly iconClass: string;
+}
+
+export function Accordion(props: AccordionProps<AccordionStyle>): ReactElement | null {
+    const styles = flattenStyles(defaultAccordionStyle, props.style);
+    const [isExpanded, setIsExpanded] = useState(!props.showHeader || props.collapsible === "collapsibleYesExpanded");
+
+    const renderIcon = () => {
+        const { iconCollapsed, iconExpanded } = props;
+        const customIconsConfigured = iconCollapsed && iconCollapsed.value && iconExpanded && iconExpanded.value;
```

Comment:
I'd use the `?.` operator here too, or even inline this as `customIconSource && customExpandedIconSource` below.